platformio_override_sample.ini

```diff
@@ -55,13 +55,13 @@ build_flags                 = ${core.build_flags}
 ; *** If you try with earlier builds a serial erase and flash is probably needed
 ;
 ; Build variant 1MB = 1MB firmware no filesystem (default)
-;board_build.ldscript        = eagle.flash.1m.ld
-; Build variant 2MB = 1MB firmware, +744k OTA, 256k filesystem (Zigbee Bridge, most Shelly devices)
-;board_build.ldscript        = eagle.flash.2m256.ld
-; Build variant 4MB = 1MB firmware, +1MB OTA, 2MB filesystem (WEMOS D1 Mini, NodeMCU, Sonoff POW)
-;board_build.ldscript        = eagle.flash.4m2m.ld
-; Build variant 16MB = 1MB firmware, +1MB OTA, 14MB filesystem (WEMOS D1 Mini pro, Ledunia (=32MB))
-;board_build.ldscript        = eagle.flash.16m14m.ld
+;board                       = esp8266_1M
+; Build variant 2MB = 1MB firmware, 1MB filesystem (most Shelly devices)
+;board                       = esp8266_2M1M
+; Build variant 4MB = 1MB firmware, 3MB filesystem (WEMOS D1 Mini, NodeMCU, Sonoff POW)
+;board                       = esp8266_4M3M
+; Build variant 4MB = 1MB firmware, 1MB OTA, 2MB filesystem (WEMOS D1 Mini, NodeMCU, Sonoff POW)
+;board                       = esp8266_4M2M
 

```

Comment:
Why were these partition changes made?

The goal of a bigger OTA space was that OTA uploads would go easier. Why do we need that much filesystem on a ESP8266?